install/kubernetes/helm/subcharts/security/templates/enable-mesh-mtls.yaml

```diff
@@ -43,7 +47,6 @@ apiVersion: networking.istio.io/v1alpha3
 kind: DestinationRule
 metadata:
   name: "api-server"
-  namespace: {{ .Release.Namespace }}
```

Comment:
why remove this line rather than doing same as above ? 
will apps in default ns pickup this DR instead of [istio-policy DR](https://github.com/istio/istio/blob/release-1.1/install/kubernetes/helm/subcharts/mixer/templates/config.yaml#L974) in istio-system when talking to mixer ?